hermes-console-vue/json-server/server.js

```diff
@@ -0,0 +1,27 @@
+const topics = require('./topics.json');
+const subscriptions = require('./subscriptions.json');
```

Comment:
When you will migrate to typescript use imports instead of 'require' function

Reply:
server.js was removed